database/factories/CountryFactory.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Model Factories
+|--------------------------------------------------------------------------
+|
+| Here you may define all of your model factories. Model factories give
+| you a convenient way to create models for testing and seeding your
+| database. Just tell the factory how a default model should look.
+|
+*/
+
+$factory->define(App\Models\Country::class, function (Faker\Generator $faker) {
+    $acronym = $faker->unique()->countryCode;
+
+    while (App\Models\Country::where('acronym', $acronym)->count() > 0 || strlen($acronym) < 2) {
```

Comment:
a bunch of users each with its own countries? Or just filling up the countries and the number is off by one?